scripts/test-verify-support-playbook-break-glass-rehearsal.sh

```
@@ -0,0 +1,281 @@
+  "detector output is authoritative" \
+  "detector activation is approved by break-glass" \
+  "assistant output is authoritative" \
+  "rollback acceptance is approved" \
+  "rollback acceptance is authoritative"; do
+  assert_rejects_forbidden_playbook_phrase "${forbidden_phrase}"
+done
+
+assert_rejects_workstation_path "$(printf '%sUsers/example/private/support-note.md' '/')"
+assert_rejects_workstation_path "$(printf 'C:%sUsers%sexample%sprivate%ssupport-note.md' '/' '/' '/' '/')"
+assert_rejects_workstation_path "$(printf 'C:%sUSERS%sexample%sprivate%ssupport-note.md' '/' '/' '/' '/')"
+assert_rejects_workstation_path "$(printf 'C:%bUsers%bexample%bprivate%bsupport-note.md' '\\' '\\' '\\' '\\')"
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)